base/statistics.jl

```diff
@@ -328,7 +330,7 @@ unscaled_covzm(x::AbstractVector{<:Number})    = sum(abs2, x)
 unscaled_covzm(x::AbstractVector)              = sum(t -> t*t', x)
 unscaled_covzm(x::AbstractMatrix, vardim::Int) = (vardim == 1 ? _conj(x'x) : x * x')
 
-unscaled_covzm(x::AbstractVector, y::AbstractVector) = dot(y, x)
+unscaled_covzm(x::AbstractVector, y::AbstractVector) = sum(conj(y[i])*x[i] for i in eachindex(y, x))
```

Comment:
The second place where a `LinearAlgebra` owned function is used. This should be ok for now...

Reply:
This is not equivalent for all element types. Should we move statistics to stdlib first?

Reply:
This indeed turned out to be a problem in some cases. See https://github.com/JuliaLang/Statistics.jl/pull/85.